src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/validation/FormAValidator.java

```diff
@@ -65,23 +72,43 @@
                 "ApplyToVary",
                 "ApplyToDischargePeriodicalPaymentOrder"
         ));
-        ALLOWED_VALUES_PER_FIELD.put(APPLYING_FOR_CONSENT_ORDER, asList("Yes"));
+        ALLOWED_VALUES_PER_FIELD.put(APPLYING_FOR_CONSENT_ORDER, Collections.singletonList("Yes"));
         ALLOWED_VALUES_PER_FIELD.put(DIVORCE_STAGE_REACHED, asList("Decree Nisi", "Decree Absolute"));
-        ALLOWED_VALUES_PER_FIELD.put("ApplicantRepresented", asList(
+        ALLOWED_VALUES_PER_FIELD.put(APPLICANT_REPRESENTED, asList(
             "I am not represented by a solicitor in these proceedings",
             "I am not represented by a solicitor in these proceedings but am receiving advice from a solicitor",
             "I am represented by a solicitor in these proceedings, who has signed Section 5"
         ));
+
+        ALLOWED_VALUES_PER_FIELD.put(ORDER_FOR_CHILDREN, asList(
+            "there is a written agreement made before 5 April 1993 about maintenance for the benefit of children",
+            "there is a written agreement made on or after 5 April 1993 about maintenance for the benefit of children",
+            "there is no agreement, but the applicant is applying for payments"
+        ));
+        ALLOWED_VALUES_PER_FIELD.put(ORDER_FOR_CHILDREN_NO_AGREEMENT, asList(
+            "for a stepchild or stepchildren",
+            "in addition to child support maintenance already paid under a Child Support Agency assessment",
+            "to meet expenses arising from a child’s disability",
+            "to meet expenses incurred by a child in being educated or training for work",
+            "when either the child or the person with care of the child or"
+                + " the absent parent of the child is not habitually resident in the United Kingdom"
+        ));
+        ALLOWED_VALUES_PER_FIELD.put(CHILD_SUPPORT_AGENCY_CALCULATION_MADE, asList(
+            "Yes",
+            "No",
+            EMPTY_STRING
+        ));
     }
-    
+
     @Override
     protected Map<String, List<String>> getAllowedValuesPerField() {
         return ALLOWED_VALUES_PER_FIELD;
     }
 
     @Override
     protected List<String> runPostProcessingValidation(Map<String, String> fieldsMap) {
-        List<String> errorMessages = Stream.of(
```

Comment:
I don't know, I actually like it says `errorMessages` as it tells me what this thing returns